src/modules/Search/Search.js

```diff
@@ -508,11 +516,12 @@ export default class Search extends Component {
   // ----------------------------------------
 
   renderSearchInput = () => {
-    const { icon, placeholder } = this.props
+    const { icon, placeholder, input } = this.props
     const { value } = this.state
 
     return (
       <Input
+        fluid={input.fluid}
```

Comment:
This should use our shorthand `Input.create` method.  Our shorthand methods allow creating elements from strings, numbers, props objects, and other elements.  This way, the user has more flexible control for defining the search input.  We can also pass some default props to use with the search input:

```jsx
return Input.create(input, {
  value,
  placeholder,
  onBlur: this.handleBlur,
  onChange: this.handleSearchChange,
  onFocus: this.handleFocus,
  onClick: this.handleInputClick,
  input: { className: 'prompt', tabIndex: '0', autoComplete: 'off' },
  icon,
})
```